11.hello-dapr-slim/README.md

```diff
@@ -0,0 +1,148 @@
+# Hello World (Slim Init, without docker dependency)
+
+This tutorial will demonstrate how to get Dapr running locally on your machine without docker. You will be deploying a Node.js app that subscribes to order messages demonstrating service invocation capability of Dapr. The following architecture diagram illustrates the components that make up this sample: 
+
+![Architecture Diagram](./img/Architecture_DiagramA.png)
+
+## Prerequisites
+This sample requires you to have the following installed on your machine:
+- [Node.js version 8 or greater](https://nodejs.org/en/) 
+- [Postman](https://www.getpostman.com/) [Optional]
+
+## Step 1 - Setup Dapr (Slim Init)
+
+Follow [instructions](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#environment-setup) to download and install the Dapr CLI and initialize Dapr.
```

Comment:
now that we have instructions for self-hosted mode can we use it instead? Or we can change this later once docs PR is merged.

..../environment-setup.md#environment-setup ---> /environment-setup.md#installing-dapr-in-self-hosted-mode

Reply:
i can create a separate PR for doc updates if needed.